CONTRIBUTING.md

```diff
@@ -355,7 +355,7 @@ conda install -c conda-forge pre-commit
 pip install pre-commit
 ```
 
-Then, run pre-commit hooks before committing your code. This wil reformat the stagged files:
+Then, run pre-commit hooks before committing your code. This will reformat the stagged files:
```

Comment:
Thanks for the contribution! Looks like there's yet another typo on the same line.
```suggestion
Then, run pre-commit hooks before committing your code. This will reformat the staged files:
```